### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by OLIPLICHE**
- I created a new branch
- I worked on mobile-menu
- I set up linters
- I used javascript to make my navigation and menu dynamics 
- I fixed linters errors
- I  added descriptive readme

  **Change asked for my first review request:** 

- I updated the demo live link in my readme file
- I added a link for the issue on my readme file
- I fixed the desktop menu alignment
- I changed my folder path

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+- I used JavaScript to manipulate DOM elements.
+- I used JavaScript events.
+
+![screenshot](./image/demo.PNG)
+
+ 
+
+## Live Demo
+
+[Live Demo Link](https://codecaiine.github.io/microverse-javascript/#contact)
```
Comment (arslanbisharat) [inline README.md]: - [x] Please update the link here as this link refers to the contact section.

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+- GitHub: [@codecaiine](https://github.com/codecaiine)
+- Twitter: [@yannicknaka](https://twitter.com/yannicknaka)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yannick-no%C3%ABl-aka/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (arslanbisharat) [inline README.md:55]: - [x] Please add a proper issue page link. You should always try to make your project Readme perfect.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"></script>
+    <link href="css/styles.css" rel="stylesheet">
```
Comment (arslanbisharat) [inline index.html]: - [x] The CSS file is not linked properly. Please add `./ `before the path to specify the location of the file.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+            <div class="logo">
+                <a href="#">My Logo</a>
+            </div>
+            <div class="openMenu"><i class="fa fa-bars"></i></div>
+            <div class="myMenu">
+                <ul class="nav-list">
+                    <li><a href="#portfolio" class="list-item">Portfolio</a></li>
+                    <li><a href="#about" class="list-item">About</a></li>
+                    <li><a href="#contact" class="list-item">Contact</a></li>
+                </ul>
```
Comment (arslanbisharat) [inline index.html:32]: [ ] While this project looks just like the given design but on the desktop mode it's giving us the menu items in vertical line while it should be horizontal. On desktop mode, the hamburger should be gone. 
 
Capture

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x]  Always try to add descriptive commit messages. I can see some of the commit messages are not descriptive. 
Capture

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by codecaiine** [comment]
demo


demooo

File: index.html (hunk 4 above)
Comment (Davidosky007) [reply index.html:32]: * done 👍

**Comment by Davidosky007** [comment]
# Hi @codecaiine 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**